src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
+import { title } from 'process';
+import { Article } from './components/Article';
```

Comment:
you exported it as default
```suggestion
import Article from './components/Article';
```